### PR TITLE
added InvertedConsumer

### PR DESCRIPTION
**Pull request by omri00**

### CODE REVIEW

**Comment by TomerSitton** [comment]
testing required

File: src/com/spikes2212/utils/InvertedConsumer.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package com.spikes2212.utils;
+
+import java.util.function.Consumer;
+
```
Comment (TomerSitton) [inline src/com/spikes2212/utils/InvertedConsumer.java]: do you think this should be in generic.utils or only in utils?

Comment (omri00) [reply src/com/spikes2212/utils/InvertedConsumer.java]: in think it should be in subsytem's utils

Comment (TomerSitton) [reply src/com/spikes2212/utils/InvertedConsumer.java]: so move it there than :)

**Pull request merged**